apps/frontend/src/libs/components/link/link.tsx

```diff
@@ -1,15 +1,23 @@
+import clsx from "clsx";
 import { NavLink } from "react-router-dom";
 
 import { type AppRoute } from "~/libs/enums/enums.js";
 import { type ValueOf } from "~/libs/types/types.js";
 
 type Properties = {
 	children: React.ReactNode;
+	className?: string | undefined;
```

Comment:
Lets remove "| undefined" because optionality operator (?) already assumes that the property may be absent

Reply:
I included undefined because styles may come from a CSS module, which can be undefined, and TypeScript won't allow me to remove the undefined type.

Reply:
Statement `className: string | undefined;` is the same as just `className?: string;`. Optionality operator ('?' after className) already means that this prop can be undefined, so setting the `undefined` type explicitly is redundant.
If you leave `className?: string;` and hover the cursor over the 'className' you'll see in your VSCode that its' type is `string | undefined`

Reply:
Yes, thanks for the info. I'll fix it on my pull request.